src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java

```diff
@@ -365,6 +366,20 @@ public Response datasetAutocomplete(@Auth AuthUser authUser, @PathParam("partial
         }
     }
 
+    @GET
+    @Path("/search")
+    @Produces("application/json")
```

Comment:
Not for this PR, but probably a tech debt PR. We really should use a constant here throughout the app.